src/wasm/Dockerfile

```diff
@@ -443,11 +467,17 @@ RUN if [ "$WITH_VRZNO" = "yes" ]; \
 RUN if [ "$WITH_MYSQL" = "yes" ]; then \
         echo -n ' --enable-mysql --enable-pdo --with-mysql=mysqlnd --with-mysqli=mysqlnd --with-pdo-mysql=mysqlnd ' >> /root/.php-configure-flags; \
     fi
+COPY --from=emscripten-curl /root/lib /root/lib-curl
+RUN if [ "$WITH_CURL" = "yes" ]; then \
+        bash /root/copy-lib.sh lib-curl && \
+        echo -n ' --with-curl ' >> /root/.php-configure-flags; \
+    fi
 
 # Build the patched PHP
 WORKDIR /root/php-src
 RUN source /root/emsdk/emsdk_env.sh && \
     emconfigure ./configure \
+    CPPFLAGS="-I/root/lib/include " \
```

Comment:
failed here.

checking `curl_easy_perform`,  probably build option is missing? 🤔